src/index.ts

```diff
@@ -632,6 +632,7 @@ process.on('SIGINT', async () => {
 
   em.emit(events.application.terminated)
 
+  // eslint-disable-next-line no-process-exit
```

Comment:
While we're fixing code, is it better to throw here instead rather than disable the warning?

reference: https://eslint.org/docs/latest/rules/no-process-exit

Reply:
If the code is intended to stop the process and exit, disable the rule is a better option.